src/internal/symbol/observable.ts

```diff
@@ -1,32 +1,27 @@
 import { root } from '../util/root';
 
-export function getSymbolObservable(context: { Symbol: SymbolConstructor; }): symbol {
-  let $$observable: symbol;
-  let Symbol = context.Symbol;
-
-  if (typeof Symbol === 'function') {
-    if (Symbol.observable) {
-      $$observable = Symbol.observable;
-    } else {
-      $$observable = Symbol('observable');
-      Symbol.observable = $$observable;
-    }
-  } else {
-    $$observable = <any>'@@observable';
+/** Symbol.observable addition */
+/* Note: This will add Symbol.observable globally for all TypeScript users,
+  however, we are no longer polyfilling Symbol.observable */
+declare global {
+  interface SymbolConstructor {
+    observable: symbol;
   }
-
-  return $$observable;
 }
 
-export const observable = getSymbolObservable(root);
+/* NOTE: Warning users that they don't have a Symbol.observable
+   polyfill. It's not something we'd throw on, because it's not
+   really *required*, however, they should be warned to give them a
+   clue as to why they might be seeing errors when trying to convert
+   ObservableLikes to Observables */
+if (!Symbol || !Symbol.observable) {
```

Comment:
This will need to do `typeof Symbol` because otherwise it still will do `Symbol is not defined`. See #3394 for reference example